src/Control/Monad/Eff/Key.purs

```diff
@@ -0,0 +1,36 @@
+module Control.Monad.Eff.Key where
```

Comment:
Perhaps should add an export list here to hide `parsePlatform`, or define `parsePlatform` in a `where` for `getPlatform`?